src/AnimalShelter.php

```diff
@@ -919,20 +1121,6 @@ public function numberOfEmployees($numberOfEmployees)
         return $this->setProperty('numberOfEmployees', $numberOfEmployees);
     }
 
-    /**
```

Comment:
This type is an organization only making offers but not getting offered.
https://schema.org/AnimalShelter